dm/pkg/checker/privilege.go

```diff
@@ -196,15 +236,34 @@ func verifyPrivileges(result *Result, grants []string, expectedGrants map[mysql.
 	}
 
 	if len(lackGrants) != 0 {
-		lackGrantsStr := make([]string, 0, len(lackGrants))
-		for g := range lackGrants {
-			lackGrantsStr = append(lackGrantsStr, mysql.Priv2Str[g])
+		var buffer bytes.Buffer
+		for g, tableMap := range lackGrants {
```

Comment:
hard to know whan happen here, could you please add some comments?

Reply:
it's should be `r`, `priv` or something rather than `g` for grant. Grant is a SQL 🤔